### PR TITLE
html-forms - Roland Estropia Jr.

### PR DESCRIPTION
**Pull request by restropiajr**
html-forms-1

### CODE REVIEW

File: html-forms/html-forms/html-forms-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+- Give five examples of form control elements.
+  Five examples of form control elements are:
+
+  1. text
+  2. textarea
+  3. select
+  4. email
+  5. radio
+
```
Comment (shawnkost) [inline html-forms/html-forms/html-forms-quiz-notes.md]: Some of these are correct and some of these aren't. `email` for example is a value for the `type` attribute and not an actual element itself. Think of actual element names like `label`

File: html-forms/html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>html-forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+
+    <form>
+      <p>Email</p>
+      <label for="email"></label>
+      <input type="email" name="email" id="email" />
```
Comment (shawnkost) [inline html-forms/html-forms/index.html]: There should be text inside your `label` on line 14. I would just remove your `p` element and throw that text inside the `label`

File: html-forms/html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+      <p>Why is coding cool?</p>
+      <label for="questionOne"></label>
+      <select name="why-coding-is-cool-select" id="questionOne">
```
Comment (shawnkost) [inline html-forms/html-forms/index.html]: There should be text inside your `label`.

File: html-forms/html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+      <label for="questionTwo">Sign me up!</label>
+
+      <p>Comments</p>
+      <label for="comments-textarea">
+        <textarea
+          name="comments"
+          id="comments-textarea"
+          cols="30"
+          rows="10"></textarea>
+      </label>
```
Comment (shawnkost) [inline html-forms/html-forms/index.html]: You can definitely nest like this, but you still need to have some sort of text associated with the `label`. You can get rid of the `p` on line 42, and throw that text inside the `label`

File: html-forms/html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+      <label for="JavaScript">JavaScript</label>
+
+      <p>Would you like to sign up for more information?</p>
+      <input type="checkbox" name="subscribe" id="questionTwo" value="yes" />
+      <label for="questionTwo">Sign me up!</label>
+
+      <p>Comments</p>
+      <label for="comments">
+        <textarea name="comments" id="comments" cols="30" rows="10"></textarea>
+      </label>
```
Comment (shawnkost) [inline html-forms/html-forms/index.html]: You can definitely nest like this, but your `label` still needs some text associated with it. You can remove the `p` element on line 42, and throw that text inside the `label`

**Review by shawnkost — CHANGES_REQUESTED** [review]
A few changes here Roland!

**Comment by restropiajr** [comment]
html-forms-1

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**